action.yml

```diff
@@ -4,10 +4,19 @@ author: 'GitHub'
 inputs: 
   name:
     description: 'Artifact name'
-    required: false
+    default: 'artifact'
   path:
     description: 'A file, directory or wildcard pattern that describes what to upload'
     required: true
+  if-no-files-found:
```

Comment:
My biggest question/concern is what should this input be? 🤔I really can't think of a good name so any ideas would be great

Reply:
I though about this a little bit more, `if-empty` or `if-empty-artifact` are some other options

Reply:
those all sound reasonable, or "if-missing"

Reply:
I think "ignore" would be clearer and more natural than "suppress" (from user's point of view they're ignoring a situation/condition, not suppressing an exception)

Reply:
I like `ignore` a lot more compared to `suppress`. Thanks for the suggestion 😃

Reply:
Sticking with `if-no-files-found` after some internal discussions. A wee bit verbose, but it's clear as to the intention